packages/macros/src/attribute/with_components/components.rs

```diff
@@ -119,6 +123,15 @@ impl AllowedComponents {
                 has_immutable_config: false,
                 internal_impls: vec!["InternalImpl"],
             },
+            AllowedComponents::AccessControlDefaultAdminRules => ComponentInfo {
+                name: "AccessControlDefaultAdminRulesComponent",
+                path: "openzeppelin_access::accesscontrol::extensions::AccessControlDefaultAdminRulesComponent",
+                storage: "access_control_DAR",
```

Comment:
```suggestion
                storage: "access_control_dar",
```

For consistency with our mocks.